dependabot-auto-merge/package.json

```diff
@@ -24,11 +24,14 @@
     "@babel/core": "7.20.12",
     "@babel/preset-env": "7.20.2",
     "@babel/preset-typescript": "7.18.6",
+    "@faker-js/faker": "^7.6.0",
     "@types/jest": "29.4.0",
     "@types/node": "18.11.18",
     "babel-jest": "29.4.2",
+    "babel-plugin-explicit-exports-references": "^1.0.2",
     "esbuild": "0.16.16",
     "jest": "29.4.2",
+    "type-fest": "^3.5.6",
```

Comment:
Very useful for doing partial mocks - `DeepPartial` is what I mainly use and it's hard to warehouse it under our repo instead of importing the whole of `type-fest`.